frontend/src/libs/components/business-card/business-card.tsx

```diff
@@ -0,0 +1,39 @@
+import rocket from '~/assets/img/welcome-page/rocket.png';
```

Comment:
Let's use svg's? (both rocket and customer)

Reply:
I think, It's better to leave these images like img, because if I understand correct in design it's really png, not svg format.
It doesn't look like svg code:
![image](https://github.com/BinaryStudioAcademy/bsa-2023-towhub/assets/110409579/b021c57a-8e3b-4f22-8809-33b0523b85d5)

Reply:
so please approve that's it'll be img tags